.gitignore

```diff
@@ -1289,3 +1289,5 @@ Node/node_modules/
 Node/package-lock.json
 PHP/**/vendor/**
 PHP/**/composer.*
+Rust/**/Cargo.lock
```

Comment:
A better way to do this is

- Go to the Edit link at the top of this file
- Add Node, PHP, and Rust to the list of ignore templates.
- Replace lines 1-1286 with the newly generated boilerplate

Reply:
I've updated the template, but I had to make some additional tweaks to enable it to find the ignorable files. I think the end result is still reasonably tidy with everything categorized.